addon/components/google-recaptcha.js

```diff
@@ -0,0 +1,104 @@
+import Component from "@ember/component";
+import layout from "../templates/components/google-recaptcha";
+import { alias } from "@ember/object/computed";
+import { isNone } from "@ember/utils";
+import { later } from "@ember/runloop";
+import { merge } from "@ember/polyfills";
+import { isPresent } from "@ember/utils";
+import { next } from "@ember/runloop";
+
+export default Component.extend({
+  /**
+   * Component based off ember-g-recaptcha ember component:
+   * https://github.com/algonauti/ember-g-recaptcha
+  */
+  layout,
+  classNames: ["g-recaptcha"],
+  sitekey: "6LeJomQUAAAAABVGp6Xk3PUZGXNWaHo3t1D7mwF3", // public captcha key
```

Comment:
same for all domains, both demo & prod, but I do think I'll add a switch in the addon config just in case there's ever a reason we need to overwrite it from a web clients